### DISCUSSION TITLE
How do we change the object that the mod is supposed to change?

### DISCUSSION THREAD

**Discussion by HorribleBee** [xivdev/Penumbra#71] [Q&A] [answered]
Maybe I'm dumb and I can't find the documentation anywhere..

But how would I change the target object of a mod? Instead of changing hairstyle 102 it would change another and soo on.

**Comment by Ottermandias** [comment, marked as answer]
You don't, at least not automatically.
You can try doing it manually, but you'd have to find all files you need to replace yourself, and probably also data from the metadata tables.
Penumbra is primarily about mod consumption, changing the target is about mod creation. You should probably import the mod in TT and use their feature for it - though I'm not generally opposed to ~~stealing~~ copying their solution somewhen, it's not gonna happen in the near future.

**Reply by HorribleBee** [reply]
I expected that, and I am trying to figure things out using only Penumbra.
If we could get an example on one object would be pretty cool, I could figure things out from there.
